src/inventory.cpp

```diff
@@ -729,26 +729,21 @@ void InventoryList::moveItemSomewhere(u32 i, InventoryList *dest, u32 count)
 u32 InventoryList::moveItem(u32 i, InventoryList *dest, u32 dest_i,
 		u32 count, bool swap_if_needed, bool *did_swap)
 {
-	if(this == dest && i == dest_i)
+	if (this == dest && i == dest_i)
 		return count;
 
 	// Take item from source list
-	ItemStack item1;
-	if(count == 0)
-		item1 = changeItem(i, ItemStack());
-	else
-		item1 = takeItem(i, count);
+	ItemStack item1 = count == 0 ? changeItem(i, ItemStack()) : takeItem(i, count);
```

Comment:
this doesn't look like an improvement in readability

Reply:
I consider it to be readable. I could probably shorten this to `count ?` instead of `count == 0 ?`.